firefox-ios/Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -153,7 +153,7 @@ class TelemetryWrapper: TelemetryWrapperProtocol, FeatureFlaggable {
         if let pingUploader = GleanPingUploader() {
             gleanConfig = Configuration(
                 channel: AppConstants.buildChannel.rawValue,
-                logLevel: .off,
+                logLevel: .debug,
```

Comment:
Glean logging can get chatty, just a reminder in case you don't want to leave this enabled.